workshops/pong_game/README.md

```diff
@@ -0,0 +1,213 @@
+* We called the `rectangle` function which we created before to draw **Paddle**. The first one will draw for player **A** and the second will draw for player **B**.
+* We will store the direction of `ball` in vector **aim**, which contains the *x* and *y* coordinates of the ball in each frame. Every time we update a new frame, simply we are adding the aim vector to the ball vector to get a new position.
+* Handling Collisions of **Pong Ball** is a very important part of code. So let us understand what we did next:
+    * The first **if** statement check if the ball hits the top or bottom of the screen. If it satisfies the condition, so simply it will reverse the **y** direction of the ball.
+    * Second **if** statement block check if ball hits **left** of the screen. Now, we first have to check if have hit the **paddle**. If we hit the paddle so simply **reverse** the **x** direction of the ball. If we did not hit the ball so **Game Over**.
+    * Similarly third **if** block check if the ball hits **right** of the screen. Same checks, if hit the paddle so **reverse** the direction or **Game Over**. So now we complete handling collision.
+* **ontime(draw, 50)** is a function of turtle library which will call **draw** function after each **50 milliseconds**. Simply it is serving as the **frame rate** of our game.
+     
+### 6. The Final Touch!
+
+As we mentioned earlier, this game is coded using a principle called modular programming. This means that everything we code is in functions, now we can bring it together. We can call these functions by writing the function name and the parameters it needs in parentheses. This code block will put all the pieces together:
+
+```python
+setup(420, 420, 370, 0)
+hideturtle()
+tracer(False)
+listen()
+onkey(lambda: move(1, 50), 'w')
+onkey(lambda: move(1, -50), 's')
+onkey(lambda: move(2, 50), 'i')
+onkey(lambda: move(2, -50), 'k')
+draw()
+done()
+```
+
+**Congrats! You've just made your own Pacman game! 🎉**
+
+![applause gif](https://cloud-460srrq0c.vercel.app/0applause.gif)
+
+Things to note:
+   * These are all functions we created earlier, we are just putting the pieces together. 
+   * We use `turtle's` built in `setup()` function to display screen.
+   * `hideturtle()`, `listen()`, and `tracer(False)` are all default `turtle` functions that will let us remove any unnecessary default items/features and enable us to listen for keystrokes.
+   * We are picking up the user's keystrokes by using Python's built-in `onkey()` function.
+   
+# Future Hacking 
+
+Our version of Pong Game is pretty basic. We are utilizing 2D vector graphics and Python’s turtle graphics library to draw shapes and dots on our screen.
+
+Here are some ideas which I suggest you try out:
+
+* How would you add a computer player? Try to build a bot that will turn the two-player game into a single-player game. Also how about build two bots which will play among them. Sound crazy right!
```

Comment:
```suggestion
* How cool it would be, if we add a computer player? Try to build a bot that will turn the two-player game into a single-player game. Also what if we build two bots which will play among themselves. Sounds crazy right!
```